### PR TITLE
fix typechecking (String => string)

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

**Comment by danbeam** [comment]
[LGTM!](http://gifrific.com/wp-content/uploads/2012/04/Tina-Fey-giving-herself-high-five.gif)

File: iron-collapse.html [hunk 1]
```diff
@@ -176,7 +176,7 @@
 
     /**
      * Updates the size of the element.
-     * @param {!String} size The new value for `maxWidth`/`maxHeight` as css property value, usually `auto` or `0px`.
+     * @param {string} size The new value for `maxWidth`/`maxHeight` as css property value, usually `auto` or `0px`.
```
Comment (valdrinkoshi) [inline iron-collapse.html]: This should be a non-null string `{!string}`

Comment (rictic) [inline iron-collapse.html]: primitives like `string` and `number` are non-null by default

Comment (valdrinkoshi) [inline iron-collapse.html]: TIL! 7 bytes saved 😁

**Comment by valdrinkoshi** [comment]
LGTM! merging this

**Comment by valdrinkoshi** [comment]
Thanks for the contribution @rictic 👍

**Pull request merged**